Documentation: Alternative Team Sync Wording (#23960)
grafana/grafana: docs/sources/auth/team-sync.md

# Team sync

> With Team Sync it's possible to setup synchronization between your auth providers teams and teams in Grafana. This enables LDAP or GitHub OAuth users which are members

Reviewer: ```suggestion
With Team Sync, you can set up synchronization between your auth provider's teams and teams in Grafana. This enables LDAP or GitHub OAuth users which are members
```
